src/main/py/commission/constraint.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+"""
+Module to manage constraint class
+
+Constraint class allows us to add constraints to an artwork
+"""
+
+import string
+
+
+class Constraint:
+    """
+    Class to manage constraint
+    """
+
+    def __init__(self, color: tuple[int, int, int, int], line_type: string):
+        """
+        Initializes an instance of the Constraint class.
+        - color (tuple(int, int, int, int)): color constraint.
```

Comment:
Please doc the meaning of the 4-tuple fields. Better yet, also use a [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple).